libraries/WebServer/src/WebServer.h

```diff
@@ -198,6 +200,7 @@ class WebServer
   int              _headerKeysCount;
   RequestArgument* _currentHeaders;
   size_t           _contentLength;
+  int              _clientContentLength;	// "Content-Length" from header of incoming POST or GET request
```

Comment:
It is necessary to initialize _clientContentLength to 0 in the WebServer constructor.
https://github.com/espressif/arduino-esp32/blob/master/libraries/WebServer/src/WebServer.cpp#L42
https://github.com/espressif/arduino-esp32/blob/master/libraries/WebServer/src/WebServer.cpp#L65

Reply:
Added class initializers per request, should be good now.